### ISSUE TITLE
Switch from Moment to Luxon

### ISSUE THREAD

**Issue by tibetsprague** [Hylozoic/hylo-evo#1346]

**Comment by lorenjohnson** [comment]
Yes! We should do already have TZ support with moment-tz but definitely a fan of moving to Luxon for the smaller bundle and it sounds like it may be necessary for the internationalization. It's a thing we need to do on all platform at once. The dependencies need to upgrade in the backend, Web and App, and then `peerDependencies` needs to update on shared. They should all be version aligned. 

Also: Much of the internationalization stuff should be put into shared as well, so all the more reason it would be best to figure-out how to best get that into a monorepo setup with `hylo-evo` sooner than later...
